### PR TITLE
Add Flake8 2.6.2 and its dependencies

### PR DESCRIPTION
**Pull request by sigmavirus24**
cc @jakirkham

### CODE REVIEW

**Comment by sigmavirus24** [comment]
Thank you @conda-forge-linter ;)

File: recipes/flake8/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - mccabe
+    - pycodestyle
+    - pyflakes
```
Comment (jakirkham) [inline recipes/flake8/meta.yaml]: Guessing these need to be run requirements.

File: recipes/flake8/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  run:
+    - python
+
+test:
+  imports:
+    - {{ name }}
+
+about:
+  home: http://flake8.pycqa.org/
+  license: Expat
```
Comment (jakirkham) [inline recipes/flake8/meta.yaml]: Maybe add MIT here? Not everyone may know what this is referring to.

**Comment by jakirkham** [comment]
Thanks, @sigmavirus24. This is awesome.

File: recipes/flake8/meta.yaml (hunk 1 above)
Comment (sigmavirus24) [inline recipes/flake8/meta.yaml]: Gah, wrong place. Will fix later.

**Comment by sigmavirus24** [comment]
Ready for a second round of review @jakirkham

File: recipes/flake8/meta.yaml (hunk 1 above)
Comment (jakirkham) [inline recipes/flake8/meta.yaml]: Fixed.

File: recipes/mccabe/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
```
Comment (jakirkham) [inline recipes/mccabe/meta.yaml]: Looks like we need `pytest-runner`?

**Comment by jakirkham** [comment]
Sorry, I totally missed this before, but we [pycodestyle]( https://github.com/conda-forge/pycodestyle-feedstock ) already. Would you like to be added, @sigmavirus24?

File: recipes/mccabe/meta.yaml (hunk 3 above)
Comment (sigmavirus24) [inline recipes/mccabe/meta.yaml]: Huh. Forgot that was a thing I did with mccabe

Comment (sigmavirus24) [inline recipes/mccabe/meta.yaml]: Fixed

**Comment by jakirkham** [comment]
Not a big deal either way, but it seems to be hanging on to `elasticsearch`. Might want to try rebasing/merging with `master` to get rid of it.

**Comment by sigmavirus24** [comment]
Rebased.

**Comment by jakirkham** [comment]
LGTM. Thanks @sigmavirus24.

**Pull request merged**